harmonica/forward/point_mass.py

```diff
@@ -0,0 +1,292 @@
+"""
+Forward modelling for point masses
+"""
+import numpy as np
+from numba import jit
+
+from ..constants import GRAVITATIONAL_CONST
+
+
+def point_mass_gravity(coordinates, point_mass, mass, field, dtype="float64"):
+    """
+    Compute gravitational fields of a point mass on spherical coordinates.
+
+    Parameters
+    ----------
+    coordinates : list or array
+        List or array containing `longitude`, `latitude` and `radius` of computation
+        points defined on a spherical geocentric coordinate system.
+        Both `longitude` and `latitude` should be in degrees and `radius` in meters.
+    point_mass : list or array
```

Comment:
Maybe we could use just `point` as it sounds weird reading `point_mass, mass`.

Reply:
:+1: you're right. The idea that we are computing the gravitational field from a point mass is very clear on the function name. So we can name its coordinates just `point`.